.github/workflows/tests.yaml

```diff
@@ -4,18 +4,18 @@ on:
   pull_request:
     types: [opened, synchronize]
     paths:
```

Comment:
I do recommend removing these paths to test to make sure things look good then readd them.